modules/LocalStorage.js

```diff
@@ -23,8 +23,9 @@ class LocalStore {
 
   static removeBook(id) {
```

Comment:
- [ ] [OPTIONAL] You can create a module for every action, like `addBook.js`, `RemoveBook.js`... this will divide your code into smaller components